spring-cloud-dataflow-docs/src/main/asciidoc/configuration-kubernetes.adoc

```diff
@@ -839,6 +863,15 @@ deployer.<application>.kubernetes.cron.ttlSecondsAfterFinished=86400
 ----
 ====
 
+The following example shows how you can individually configure application jobs:
+
+====
+[source,options=nowrap]
+----
+deployer.<application>.kubernetes.ttlSecondsAfterFinished=86400
```

Comment:
What is this related to?

Reply:
There was an addition done a while ago for cron.ttlSecondsAfterFinished and this was added but not documented. #5406

Reply:
Got it. Even though it is small I would probably put this in its own commit in this PR for later find/perusal. I think it is not significant enough to change now though.